src/App.jsx

```diff
@@ -68,7 +68,7 @@ const App = () => {
                 <Route exact path="/collections/:collectionId" component={Collection} />
                 <Route exact path="/collections/:collectionId/edit" component={EditCollection} />
                 <Route exact path="/collections/add" component={AddCollection} />
```

Comment:
Do we need to change the `/collection/add` endpoint as well, since it might conflict with `/collection/:collectionId`?

Reply:
It doesn't seem to interfere though. What should we change it to?